samples/snippets/update_intent.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud.dialogflow_v2 import IntentsClient
```

Comment:
Should this be:

```python
from google.cloud.dialogflow import IntentsClient
```
In the current code state they will be equivalent, but IIUC this change will allow this snippet to still work if/when the api has a major revision bump.